examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -515,18 +517,19 @@ void saveWifiConfigCallback(void) {
   flagSaveWifiConfig = true;
 }
 
-// Forcibly mount the SPIFFS. Formatting the SPIFFS if needed.
+// Forcibly mount the FILESYSTEM. Formatting the FILESYSTEM if needed.
 //
 // Returns:
 //   A boolean indicating success or failure.
 bool mountSpiffs(void) {
-  debug("Mounting SPIFFS...");
-  if (SPIFFS.begin()) return true;  // We mounted it okay.
+  debug("Mounting FILESYSTEM...");
+  if (FILESYSTEM.begin()) return true;  // We mounted it okay.
   // We failed the first time.
-  debug("Failed to mount SPIFFS!\nFormatting SPIFFS and trying again...");
-  SPIFFS.format();
-  if (!SPIFFS.begin()) {  // Did we fail?
-    debug("DANGER: Failed to mount SPIFFS even after formatting!");
+  debug("Failed to mount FILESYSTEM!\n"
+        "Formatting FILESYSTEM and trying again...");
```

Comment:
```
  debug("Failed to mount " FILESYSTEM "!\n"
        "Formatting " FILESYSTEM " and trying again...");
```
Use that instead, and it should report if it is SPIFFS or LittleFS etc.

Reply:
Ditto.